modules/apps/portal/portal-monitoring/src/main/java/com/liferay/portal/monitoring/internal/statistics/jmx/ActionRequestPortletManager.java

```diff
@@ -33,10 +32,12 @@ public ActionRequestPortletManager() throws NotCompliantMBeanException {
 
 	@Override
 	protected PortletSummaryStatistics getPortletSummaryStatistics() {
-		return _actionRequestSummaryStatistics;
+		return _portletSummaryStatistics;
 	}
 
-	@Reference
-	private ActionRequestSummaryStatistics _actionRequestSummaryStatistics;
+	@Reference(
+		target = "(component.name=com.liferay.portal.monitoring.internal.statistics.portlet.ActionRequestSummaryStatistics)"
+	)
+	private PortletSummaryStatistics _portletSummaryStatistics;
```

Comment:
Hey @Valmir-unicap, rename variable to `_actionRequestSummaryStatistics`. Please apply to other cases. Thx